src/components/package/edit-custom/custom-package-edit.js

```diff
@@ -220,17 +222,11 @@ class CustomPackageEdit extends Component {
                 <DetailsViewSection
                   label={intl.formatMessage({ id: 'ui-eholdings.label.holdingStatus' })}
                 >
-                  <label
-                    data-test-eholdings-package-details-selected
-                    htmlFor="custom-package-details-toggle-switch"
-                  >
-                    <h4>
-                      {packageSelected ?
-                        (<FormattedMessage id="ui-eholdings.selected" />) :
-                        (<FormattedMessage id="ui-eholdings.notSelected" />)
-                      }
-                    </h4>
-                  </label>
+                  <SelectionStatus
+                    model={model}
+                    isPending={false}
```

Comment:
We shouldn't need this property anymore because it's on `model` right?

Reply:
Thanks for catching @cowboyd -- just removed!